src/modules/common/components/nav-panel/nav-panel.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import classNames from 'classnames';
+import Styles from 'modules/common/components/nav-panel/nav-panel.styles';
+
+
+const NavPanel = p => (
+  <div className={Styles.NavPanel}>
+    <aside className={Styles.NavPanel__controls}>
+      {p.items && p.items.map((item, ind) => {
+        const Icon = item.iconComponent;
+        return (
+          <button
+            className={classNames(
+              {
+                [Styles.NavPanel__control]: true,
+                [Styles['NavPanel__control--active']]: item.active
+              })}
+            key={`${item.title}${ind - p.items.length}`}
+            onClick={item.onClick}
```

Comment:
Instead of passing through an `onClick` property, you can just have a method that updates the query string param for this specific nav (will also need another param) via `history.push`